sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -71,15 +85,15 @@ class SnackBar extends Component {
     ];
     if (actions != null)
       children.addAll(actions);
-
     return new SlideTransition(
```

Comment:
(side note: it'd be really nice if there were a way to avoid the syncConstructorArg boilerplate for stateful components in general.. somehow)